src/runtime/api/auth.ts

```diff
@@ -149,6 +149,10 @@ export default class Auth {
     return this.user.roles
   }
 
+  public get signedIn () {
```

Comment:
Brill, if you could also test this function that'd be great

Reply:
tests added

Reply:
added tests